openverse_catalog/dags/util/loader/sql.py

```diff
@@ -4,16 +4,16 @@
 
 from airflow.providers.postgres.hooks.postgres import PostgresHook
 from psycopg2.errors import InvalidTextRepresentation
-from util.loader import column_names as col
+from storage import column_names as col
+from util.constants import AUDIO, IMAGE
 from util.loader import provider_details as prov
 from util.loader.paths import _extract_media_type
 
 
 logger = logging.getLogger(__name__)
 
 LOAD_TABLE_NAME_STUB = "provider_data_"
-IMAGE_TABLE_NAME = "image"
-AUDIO_TABLE_NAME = "audio"
+TABLE_NAME = {AUDIO: AUDIO, IMAGE: IMAGE}
```

Comment:
This is definitely a nit so feel free to ignore, but I think this might be better as `TABLE_NAMES`. I've typically seen that convention used with collections, and it makes a bit more sense when reading `TABLE_NAMES.get("foo")`. Another alternative is `TABLE_NAME_LOOKUP` but I like that less haha

Reply:
`TABLE_NAMES.get("foo")` reads much better, agreed!